src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/Helpers/NativeMethods.Unix.vb

```diff
@@ -0,0 +1,20 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Option Strict On
+Option Explicit On
+
+Imports System.Runtime.InteropServices
+
+Namespace Microsoft.VisualBasic.CompilerServices
+
+    <ComVisible(False)>
+    Friend NotInheritable Class NativeMethods
+        Friend Shared Function MoveFileEx(
```

Comment:
Are there any callers? If not, please remove this method.

Reply:
Good catch, left over from trying to use MoveFIleEx